tests/test_e2e.py

```diff
@@ -0,0 +1,41 @@
+import optuna
+
+try:
+    # neptune-client=0.9.0+ package structure
+    import neptune.new as neptune
+except ImportError:
+    # neptune-client>=1.0.0 package structure
+    import neptune
+
+import neptune_optuna.impl as npt_utils
+
+def test_e2e():
+
+    n_trials = 5
+    base_namespace = "optuna"
+
+    run = neptune.init_run()
+    # handler = run[base_namespace]
```

Comment:
I think we want to add another test with `handler` instead of `run`?

Reply:
Yes, but that is not supported with the current codebase and here I am only adding tests to the existing code.

Reply:
Got it!